tachyon/math/finite_fields/quadratic_extension_field.h

```diff
@@ -1,20 +1,242 @@
+// Copyright 2022 arkworks contributors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.arkworks and the LICENCE-APACHE.arkworks
+// file.
+
 #ifndef TACHYON_MATH_FINITE_FIELDS_QUADRATIC_EXTENSION_FIELD_H_
 #define TACHYON_MATH_FINITE_FIELDS_QUADRATIC_EXTENSION_FIELD_H_
 
+#include "absl/strings/substitute.h"
+
+#include "tachyon/math/base/field.h"
+
```

Comment:
I think you should include the necessary headers for `std::move` and `std::ostream`
```suggestion
#include <utility>
#include <ostream>
```